.travis.yml

```diff
@@ -10,6 +10,9 @@ before_script:
 script:
   - cargo run -p ci
   - cargo run --example default
+  - if [ "${TRAVIS_RUST_VERSION}" = nightly ]; then cargo update -Z minimal-versions; fi
```

Comment:
A separate travis job is fine, but what do you mean by "just the `-Z minimal-versions` check at the start"? Just `cargo update -Z minimal-versions` is not enough.

Reply:
Ah I meant switch on some environment variable instead of the nightly version. But the script is only a few lines so we could just duplicate the script in the travis environment specifically for testing minimal versions.

Reply:
Gotcha, I'll try the switch one then.